### PR TITLE
Add a flexible set of clientMaxBodySize limits

### PR DESCRIPTION
**Pull request by raknamber1**

### CODE REVIEW

File: charts/dify/values.yaml [hunk 1]
```diff
@@ -241,6 +241,8 @@ proxy:
   customReadinessProbe: {}
   ## @param proxy.customStartupProbe Custom startupProbe that overrides the default one
   customStartupProbe: {}
+  ## @param.clientMaxBodySize Custom client_max_body_size param nginx default: 15m
```
Comment (BorisPolonsky) [inline charts/dify/values.yaml:244]: keyword `proxy` was missing. Do you mean
`## @param proxy.clientMaxBodySize Custom client_max_body_size param nginx default: 15m`

**Review by BorisPolonsky — CHANGES_REQUESTED** [review]
Minor adjustment needed in comment

**Comment by raknamber1** [comment]
@BorisPolonsky I fixed it, you can look at it again

**Review by BorisPolonsky — APPROVED** [review]

**Comment by raknamber1** [comment]
@BorisPolonsky please tell me, can we release this or do we need to wait for other changes?

**Comment by BorisPolonsky** [comment]
> @BorisPolonsky please tell me, can we release this or do we need to wait for other changes?

I'm currently busy with my work. If you need this feature ASAP you can pull from master branch and make a package of your own with `helm package` command. It's advised to maintain a temporary version tag in `Chart.yaml` for this purpose

**Pull request merged**